### ISSUE TITLE
make lint

### ISSUE THREAD

**Issue by craff** [lindig/polly#8]
make lint is not configured in the same way as lint in the worflow ?
on my machine, make lint reports no error while the workflow fails with the same code.

**Comment by craff** [comment]
needed quite a lot of time to make lint happy because of this.

**Comment by lindig** [comment]
I agree that the CI workflow is quite picky. I merged it recently from a PR and struggled with it, too. It has access to several OCaml releases, unlike the local installation. I am a bit torn whether to make the listing less struct. On the other hand, it is likely to help with submissions to Opam.

**Comment by craff** [comment]
I make lint would just do the same check, that will be ok for me. 

Le 5 août 2023 05:55:59 GMT-10:00, Christian Lindig ***@***.***> a écrit :

**Comment by edwintorok** [comment]
I think 'dune build @fmt' is missing from 'make lint' (there is an indent target in the Makefile though)

**Comment by craff** [comment]
in fact, make lint + make format is ok. Maybe make test could depend upon lint and format ?
